installing/install_config/configuring-private-cluster.adoc

```diff
@@ -5,7 +5,7 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
-After you install {product-title} version 4.2 cluster, you can set some of its core components to be private.
+After you install {product-title} version {product-version} cluster, you can set some of its core components to be private.
```

Comment:
The replace is fine. And I know it was like this before that, but it reads weird if you want to fix it while you're in there.

I'd either put "an" in front of `{product-title} `, or remove "cluster". So it either says:

```
After you install an OpenShift Container Platform version 4.4 cluster,
or
After you install OpenShift Container Platform version 4.4,
```

Reply:
Yep! Better to improve it while I'm in there. I'll fix the flow. Thanks!

Reply:
Thanks @codyhoag!